packages/editor/src/components/post-last-revision/index.js

```diff
@@ -36,8 +36,7 @@ function PostLastRevision() {
 	return (
 		<PostLastRevisionCheck>
 			<Button
-				// TODO: Switch to `true` (40px size) if possible
-				__next40pxDefaultSize={ false }
+				__next40pxDefaultSize
```

Comment:
I wasn't able to find how to test this instance in the editor — it looks like the `PostLastRevision` component is not used anymore since https://github.com/WordPress/gutenberg/pull/62323 (cc @ntsekouras ) but we're still exporting it.

In any case, I can see that [there are some style overrides in place](https://github.com/WordPress/gutenberg/blob/b0c9f4be156c505eb66f96b425543381d0d3078f/packages/editor/src/components/post-last-revision/style.scss#L1-L20) that tweak the `height`, and so passing the `__next40pxDefaultSize` prop shouldn't affect the look of this button.

For the scope of this PR, I think that we can leave things as they are.

As a future improvement, we should avoid [overriding button (and panel) styles](https://github.com/WordPress/gutenberg/blob/b0c9f4be156c505eb66f96b425543381d0d3078f/packages/editor/src/components/post-last-revision/style.scss). If we feel like we're tweaking `Button` beyond its purpose, we should consider not using `Button` at all (cc @mirka )

Reply:
Yeah, this component is not used any more in favor of `PrivatePostLastRevision` component. There was some discussions whether to remove it [here](https://github.com/WordPress/gutenberg/pull/62323/files#r1630804978) and we ended up keeping it for existing consumers.

Reply:
I think it still makes sense to remove the overrides if possible.

Reply:
Agreed — it would be good to do that as a follow up to this PR.

Reply:
A follow-up PR sounds good 👍